contrib/backport.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+set -e
+shopt -s extglob # enable extended pattern matching features
+
+
+#############
+# VARIABLES #
+#############
+
+stable_branch=$1
+commit=$2
+bkp_branch_name=$3
+bkp_branch_name_prefix=bkp
+bkp_branch=$bkp_branch_name-$bkp_branch_name_prefix
+
+
+#############
+# FUNCTIONS #
+#############
+
+git_status () {
+  if [[ $(git status --porcelain | wc -l) -gt 0 ]]; then
+    echo "It looks like you have not committed changes:"
+    echo ""
+    git status --short
+    echo ""
+    echo ""
+    echo "Press ENTER to continue or Ctrl+C to break."
+    read -r
+  fi
+}
+
+checkout () {
+  git checkout -b "$bkp_branch" origin/"$stable_branch"
```

Comment:
What do you think about using `--no-track` here? When we use `git checkout -b` with a remote branch, it'll set it up as tracking this remote branch. We don't want to push to this later on.